src/backend/vaapi.rs

```diff
@@ -38,6 +38,37 @@ use crate::y410_to_i410;
 use crate::DecodedFormat;
 use crate::Resolution;
 
+fn va_fourcc_to_string(va_fourcc: u32) -> String {
+    String::from(match va_fourcc {
+        libva::constants::VA_FOURCC_NV12 => "NV12",
+        libva::constants::VA_FOURCC_I420 => "I420",
+        libva::constants::VA_FOURCC_422H => "422H",
+        libva::constants::VA_FOURCC_444P => "444P",
+        libva::constants::VA_FOURCC_P010 => "P010",
+        libva::constants::VA_FOURCC_P012 => "P012",
+        libva::constants::VA_FOURCC_Y210 => "Y210",
+        libva::constants::VA_FOURCC_Y212 => "Y212",
+        libva::constants::VA_FOURCC_Y410 => "Y410",
+        libva::constants::VA_FOURCC_Y412 => "Y412",
+        other => return format!("unknown VA fourcc 0x{:x}", other),
```

Comment:
I guess we could also build the strings directly from the constants, since they are basically these four characters turned into a u32. Note to self to verify and do this if possible.

Reply:
Done in 6dd7bd6.